xlml/utils/metric_test.py

```diff
@@ -315,41 +315,6 @@ def test_add_airflow_metadata(self):
 
           self.assert_metric_and_dimension_equal([], [], actual_value, expected_value)
 
-  @parameterized.named_parameters(
-      (
-          "prod_scheduled_run",
-          composer_env.PROD_COMPOSER_ENV_NAME,
-          "scheduled__2023-08-07T21:03:49.181263+00:00",
-          True,
-      ),
-      (
-          "non-prod_scheduled_run",
-          composer_env.DEV_COMPOSER_ENV_NAME,
-          "scheduled__2023-08-07T21:03:49.181263+00:00",
-          False,
-      ),
-      (
-          "prod_manual_run",
-          composer_env.PROD_COMPOSER_ENV_NAME,
-          "manual__2023-08-07T21:03:49.181263+00:00",
-          False,
-      ),
-  )
-  def test_is_valid_entry(self, env_name, run_id, expected_value):
```

Comment:
Why is this test removed? Is there another test case that replaces it?

Reply:
Yeah, I removed `is_valid_entry()` on purpose. I think no matter it's scheduled or manual runs in prod/dev, all data should be in prod/dev dataset, respectively. I did not add a test for `update_dataset_name_if_needed`, and I think it's quite straightforward. HDYT?

Reply:
Going to merge this PR to unblock @raymondzouu's test. @will-cromar let's discuss offline if you have any question about my reply. Thank you!